dataactbroker/handlers/fileHandler.py

```diff
@@ -434,13 +419,24 @@ def getStatus(self):
             submission_info = {}
             submission_info["jobs"] = []
             submission_info["cgac_code"] = submission.cgac_code
-            submission_info["reporting_period_start_date"] = self.interfaces.jobDb.getStartDate(submission)
-            submission_info["reporting_period_end_date"] = self.interfaces.jobDb.getEndDate(submission)
-            submission_info["created_on"] = self.interfaces.jobDb.getFormattedDatetimeBySubmissionId(submission_id)
+            submission_info["created_on"] = submission.datetime_utc.strftime('%m/%d/%Y')
             # Include number of errors in submission
             submission_info["number_of_errors"] = submission.number_of_errors
-            submission_info["number_of_rows"] = self.interfaces.jobDb.sumNumberOfRowsForJobList(jobs)
+            submission_info["number_of_rows"] = sess.query(
+                func.sum(Job.number_of_rows)).\
+                filter_by(submission_id = submission_id).\
+                scalar() or 0
             submission_info["last_updated"] = submission.updated_at.strftime("%Y-%m-%dT%H:%M:%S")
+            # Format submission reporting date
+            if submission.is_quarter_format:
+                reporting_date = 'Q{}/{}'.format(
+                    int(submission.reporting_fiscal_period / 3), submission.reporting_fiscal_year)
+            else:
+                reporting_date = submission.reporting_start_date.strftime("%m/%Y")
+            # Broker allows submission for a single quarter or a single month,
+            # so reporting_period start and end dates reported by check_status
+            # are always equal
+            submission_info["reporting_period_start_date"] = submission_info["reporting_period_end_date"] = reporting_date
```

Comment:
That's a good question. A long time ago, broker users could actually specify submission start/end dates that included a day (_e.g._, 7/13/2015 - 8/09/2016). Thus, our models are set up to accommodate this format. 

As the broker evolved, we realized that real-world submissions will only come in 2 flavors:
* A single quarter (this is the only submission type eligible for certification and import to usaspending). The` check_status` date format for this is `Q1/2017`, for example.
* A single month (for agencies to do internal data testing). The `check_status` date format for this is `10/2016`.

With this PR, we're now depending on the front-end to send us a start/end date that are both in the same quarter (for quarterly submission) and a start/end date in the same month (for monthly submissions). That makes the actual day irrelevant. 

You can see this in the current code by looking at the reporting start/end dates being returned by the `check_status` route. 

Should I try to clarify the comment? Do you think it makes sense to add some logic/testing around the submission reporting dates to make sure we don't get anything other than a single quarter or single month?